src/modules/testimonials/testimonials.service.ts

```diff
@@ -0,0 +1,42 @@
+import { HttpStatus, Injectable, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { User } from '../user/entities/user.entity';
+import { CreateTestimonialDto } from './dto/create-testimonial.dto';
+import { Testimonial } from './entities/testimonials.entity';
+
+@Injectable()
+export class TestimonialsService {
+  constructor(
+    @InjectRepository(Testimonial)
+    private readonly testimonialRepository: Repository<Testimonial>,
+
+    @InjectRepository(User)
+    private readonly userRepository: Repository<User>
+  ) {}
+  async create(createTestimonialDto: CreateTestimonialDto, userId: string) {
```

Comment:
could you make this name more descriptive like having createTestimonial

Reply:
Renamed to createTestimonial